### PR TITLE
fixed yestheoryuploaded ping

### PR DESCRIPTION
**Pull request by hasanlatch007**
If webhooks count as bots this should not return an error and we should no longer need to use `hasRole`.
Used the test YesBot by forcing it to send a message in #yestheory-uploaded which pinged the group.

### CODE REVIEW

**Comment by hasanlatch007** [comment]
This actually might not work I checked the discord.js docs and webhooks are not logged in as bots? So `message.author.bot` might always return false, I'm not sure of this I gotta read the docs a bit better.
`if (message.webhookID)` Will have to replace it with this I think :')

**Comment by hasanlatch007** [comment]
Both `webhookID` or `message.author.bot` work but the first one will only return true if the message is from a webhook that serves our goal better.

File: src/programs/yestheory-content.ts [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
 import { ChatNames } from "../collections/chat-names";
-import { hasRole } from "../common/moderator";
 
 //keep in mind this is very temporarily basic code, until we have the new events up and ready
 const YesTheoryUploadedPing = async (message: Message) => {
-  const user = message.author.id;
-  const MemberGuild = await message.guild.members.fetch(user);
-  if (hasRole(MemberGuild, "Support")) {
-    return;
+  if (message.webhookID) {
```
Comment (geisterfurz007) [inline src/programs/yestheory-content.ts:6]: Use an early return instead.

**Review by geisterfurz007 — CHANGES_REQUESTED** [review]

**Comment by geisterfurz007** [comment]
`webhookID` was a really nice catch!

File: src/programs/yestheory-content.ts (hunk 1 above)
Comment (hasanlatch007) [reply src/programs/yestheory-content.ts]: Done :)

**Comment by hasanlatch007** [comment]
> `webhookID` was a really nice catch!

I used my big brain for once :D

**Review by geisterfurz007 — APPROVED** [review]

**Pull request merged**